app.js

```diff
@@ -1,138 +1,188 @@
+          localStorage.setItem("meet-data", google_meet_transcript_data);
+          console.log(results[0].flag, results, "value of take snapshot bool");
+          if (results[0].flag) {
+            console.log("take snapshot!! :)");
+            takeScreenShot();
+            acknowledgeSSSuccess();
+          }
+          start_fetching_meet();
+        }
+      );
+    }, 5000);
+  } else return;
   // }
 }
 
-function get_host_name(){
+function get_host_name() {
   return window.location.host;
 }
 
-start_button.addEventListener("click",async function(){
-  localStorage.setItem('images',JSON.stringify([]));
-  await chrome.tabs.executeScript(null, { code: '(' + get_host_name + ')();' }, (results) => {
-    console.log(results);
-      if(results[0] == "meet.google.com"){
+start_button.addEventListener("click", async function () {
+  localStorage.setItem("images", JSON.stringify([]));
+  await chrome.tabs.executeScript(
+    null,
+    { code: "(" + get_host_name + ")();" },
+    (results) => {
+      console.log(results);
+      if (results[0] == "meet.google.com") {
         is_google_meet = true;
-        console.log('It is a G Meet');
+        console.log("It is a G Meet");
       }
-      if(is_google_meet){
+      if (is_google_meet) {
         catch_meet = true;
-        console.log('starting recording...')
+        console.log("starting recording...");
         submit_button.style.display = "inline-block";
         start_button.style.display = "none";
         start_fetching_meet();
-      }else toggleSpeechRecognition();
-  });
+      } else toggleSpeechRecognition();
+    }
+  );
 });
 
-submit_button.addEventListener("click", function(e){
+submit_button.addEventListener("click", function (e) {
   e.preventDefault();
-  if(!is_google_meet)toggleSpeechRecognition();
-  else{
+  if (!is_google_meet) toggleSpeechRecognition();
+  else {
     recognizing = false;
-    submit_button.style.display = 'none';
-    start_button.style.display = 'inline-block';
+    submit_button.style.display = "none";
+    start_button.style.display = "inline-block";
   }
   const req = new XMLHttpRequest();
   const baseUrl = BASE_URL_VS + "/text/1";
-  const subject_name = document.getElementById('subject_name').value;
-  const lecture_name = document.getElementById('lecture_name').value;
-  const num_lines = document.getElementById('num_lines').value;
-  if(num_lines == null)num_lines = -1;
+  const subject_name = document.getElementById("subject_name").value;
+  const lecture_name = document.getElementById("lecture_name").value;
+  const num_lines = document.getElementById("num_lines").value;
+  if (num_lines == null) num_lines = -1;
   // const transcript = document.getElementById('transcript').value;
   // const transcript_data = localStorage.getItem('current_session');
-  if(is_google_meet){
+  if (is_google_meet) {
     catch_meet = false;
     transcript_data = google_meet_transcript_data;
   }
```

Comment:
Here, the transcript_data will be from the localstorage.

Change, transcipt_data = localStorage.getItem('google-meet-data')